data/campaigns/fri04.wmf/scripting/mission_thread.lua

```diff
@@ -35,8 +48,13 @@ function see_reebaud(field)
    campaign_message_box(reebaud_5)
    campaign_message_box(reebaud_6)
    campaign_message_box(reebaud_7)
-   campaign_message_box(reebaud_8)
-   if seen_amazons then campaign_message_box(reebaud_9a) else campaign_message_box(reebaud_9b) end
+   if p4.defeated then
+      amazons_defeated_by_reebaud = true
+      campaign_message_box(reebaud_8b)
+   else
```

Comment:
if the empire is not present the game ends 5 seconds after the message 8b so we might need either:
1. a longer sleep time (to simulate Reebaud's men to recover) leaving the player without anything to do specifically. 
2. A new objective to recruit a certain amount of soldiers to replenish Reebauds army 
I would like 2 best, as this could be given as second objective to defeating the empire as well.

Reply:
nice solution. like that